app/models/miq_ui_worker/runner.rb

```diff
@@ -1,3 +1,8 @@
 class MiqUiWorker::Runner < MiqWorker::Runner
   include MiqWebServerRunnerMixin
+
+  def prepare
+    super
+    MiqApache::Control.start if MiqEnvironment::Command.is_container?
```

Comment:
I thought this was going away.

Reply:
Why?

We need it to serve static assets.

Reply:
Forgot about that 👍